src/data/nav/chat.ts

```diff
@@ -129,6 +129,11 @@ export default {
           name: 'Livestream chat',
           link: '/docs/guides/chat/build-livestream',
         },
+        {
+          name: 'Replies',
+          link: '/docs/guides/chat/replies',
+          languages: ['react', 'javascript'],
```

Comment:
This isn't necessary if it's now relevant to both JS and React - the dropdown gets automatically populated and this is just an override.